locale/pt_BR/wivrn-dashboard.po

```diff
@@ -659,21 +665,38 @@ msgstr ""
 msgid "I cannot see my computer in the WiVRn app"
 msgstr "Não consigo ver meu computador no aplicativo do WiVRn"
 
-#: dashboard/qml/TroubleshootPage.qml:25 dashboard/qml/WizardPage.qml:393
+#: dashboard/qml/TroubleshootPage.qml:25
 #, kde-format
-msgid ""
-"If the server is not visible or the connection fails, check that port 5353 "
-"(UDP) and 9757 (TCP and UDP) are open in your firewall."
+msgid "If you have a firewall, make sure that port 5353/UDP is open."
 msgstr ""
-"Se o servidor não estiver visível ou a conexão falhar, verifique se a porta "
-"5353 (UDP) e 9757 (TCP e UDP) estão abertas no seu firewall."
+"Se você tem firewall, se certifique que a porta 5353/UDP esteja aberta."
 
 #: dashboard/qml/TroubleshootPage.qml:28
 #, kde-format
+msgid "Connecting remains stuck on \"Connection to ...\""
+msgstr "Conexão permanece travada em \"Conectando a ...\""
+
+#: dashboard/qml/TroubleshootPage.qml:29 dashboard/qml/TroubleshootPage.qml:33
+#, kde-format
+msgid ""
+"If you have a firewall, make sure that port 9757 is open both for TCP and "
+"UDP."
+msgstr ""
+"Se você tem firewall, certifique-se que a porta 9757 esteja aberta tanto para "
+"TCP quando "
+"UDP."
+
+#: dashboard/qml/TroubleshootPage.qml:32
+#, kde-format
+msgid "Connection fails with \"no route to host\""
+msgstr "Conexão falha com \"sem rota para o host\""
```

Comment:
If I recall correctly, system errors are not translated on headset.

Reply:
Will fix it

Reply:
done